src/components/project/list.vue

```diff
@@ -19,28 +19,34 @@ except according to the terms contained in the LICENSE file.
           @click="showModal('newProject')">
           <span class="icon-plus-circle"></span>{{ $t('action.create') }}&hellip;
         </button>
+        <project-sort v-model="sortMode"/>
       </template>
       <template #body>
-        <table class="table">
-          <thead>
-            <tr>
-              <th>{{ $t('header.name') }}</th>
-              <th>{{ $t('resource.forms') }}</th>
-              <th>{{ $t('header.lastSubmission') }}</th>
-            </tr>
-          </thead>
-          <tbody v-if="projects != null">
-            <project-row v-for="project of projects" :key="project.id"
-              :project="project"/>
-          </tbody>
-        </table>
+        <div v-if="projects != null">
+          <project-home-block v-for="project of activeProjects" :key="project.id"
+            :project="project" :sort-func="sortFunction"/>
+        </div>
         <loading :state="$store.getters.initiallyLoading(['projects'])"/>
         <p v-if="projects != null && projects.length === 0"
           class="empty-table-message">
           {{ $t('emptyTable') }}
         </p>
       </template>
     </page-section>
+    <page-section v-if="archivedProjects.length > 0">
+      <template #heading>
+        <span>{{ $t('archived') }}</span>
+      </template>
+      <template #body>
+        <div id="archived-projects">
```

Comment:
Since it's an id, it'd be great to prefix it with the name of the component: `project-list-archived` maybe.